generators/app/templates/gulp/tasks/scripts.js

```diff
@@ -1,44 +1,41 @@
 'use strict';
 
 var scriptsTask = function (gulp, plugins, config, helpers) {
+  function buildScript(watch) {
+    var props = {
+      entries: config.src.app,
+      debug: true,
+      transform: [["babelify", { "presets": ["es2015"] }]]
+    };
 
-  var customOpts = {
-    entries: config.src.app,
-    debug: true
-  };
-
-  function bundle(bundler) {
-    return bundler<% if (features.has_babel) { %>
```

Comment:
Yeaman feature detection `features.has_babel` is missing in new version. This causes build to fail, because of lack of `babel`